### DISCUSSION TITLE
new antag (cannibal)

### DISCUSSION THREAD

**Discussion by mmails** [unitystation/unitystation#7301] [Ideas]
### Description

You are a cannibal! You have been cursed with a spell by the Wizard Federation that requires you to feast on human flesh to avoid dying of hunger.

(3 tasks, probably eating high ranking personnel, such as the captain, hos, etc)

**Comment by PerfectTangent** [comment]
converting this to a discussion.

**Comment by mmails** [comment]
Anyways, their greentext could be:

1. Consume the captain.
2. Consume the HoS
3. Get on the shuttle alive!

**Reply by mmails** [reply]
Something alone these lines atleast

**Reply by AtnerNT** [reply]
hmm I feel like abit more could be done with this, rn this just seems like an antag with only assasinate objectives- but with some flavour text of eating them after. I have a few suggestions: 
1) Maybe the cannibal acts as a hardmode traitor- as due to their hunger they are ona  time limit of 10-15 minutes to complete their objeciv or they starve to death- if they are unable to eat there targets in that time- they must feast on the crew to sustain themselves before they can strike down the captain
2) They arent a traitor- but instead a result of cultists. being under the influence of cutist magic or spending too long in there presence can result in you gaining a hunger for human flesh which you must consume or die- (this could happen to anyone mid round during a cultist match, instead of being an antag one would spawm as) if we treat cannibalism as a trait or symptom a normal crew member can get- it could be also be added to the list of symptoms for diseases, a round could have a plague of cannibalism that is spreading- so as the station quarantines and science attempts to cure the infection, security must contain those effected as they run  around attempted to eat the crew all while remaining un infected themselves.

**Reply by mmails** [reply]
Sounds good, i made it at like 3 am thinking it would get buried, but glad someone fixed my shitty suggestion 😂

**Reply by mmails** [reply]
Another idea,

If they don't eat someone in the 10-15 minute timespan, they could start eating their own skin/flesh, as a way to urge them to eat, and aswell as to make them recognizable 

_You haven't eaten anything...you start unconsciously eating your own skin!_ 

They could take brute damage every 2 to 5 minutes
